### ISSUE TITLE
Can´t connect to board to flash

### ISSUE THREAD

**Issue by henmartins27** [wuxx/nanoESP32-C6#7]
Hello,
I followed the demo guide to verify my board on both Windows 11 and Linux (Ubuntu). In both cases I get the following problem:
```
Serial port /dev/ttyACM0
Connecting......................................

A fatal error occurred: Failed to connect to ESP32-C6: No serial data received.
```
On Windows I use serial port COM3.

How to fix this behavior? Thanks

**Comment by wuxx** [comment]
Hello, what is the complete command you entered

**Comment by henmartins27** [comment]
@wuxx I used the same command as the demo:

`$git clone https://github.com/espressif/esp-idf.git
$cd esp-idf && ./install.sh && . ./export.sh
$cd examples/get-started/blink
$idf.py set-target esp32c6
$idf.py -p /dev/ttyACM0 flash monitor`

I now tried running it on Arduino IDE (with the recommended config) and I faced the exact same error.

**Comment by wuxx** [comment]
Please try manually entering download mode, holding down the BOOT button to power on, and then executing the download command

**Comment by henmartins27** [comment]
@wuxx Tried this (hold down BOOT, then press RST then release both) and still has same issue. Is it possible that my board is defective?

**Comment by henmartins27** [comment]
Now I also tried on macOS (completely different computer) and found the same error...

**Comment by wuxx** [comment]
The default program is a flashing light. Can you observe it? In addition, there are two USB ports on the board. Which one did you plug in?

**Comment by henmartins27** [comment]
No, the light never turns on. I’m using the port on the left - CH343. The other one is never recognized by the computer. 

Just to note, the error is different when the board is not plugged in, so the computer is seeing something, it’s just not managing to flash it.

**Comment by wuxx** [comment]
Can you check the module to see if there is any solder joint problem?

**Comment by Bart56** [comment]
I'm plugging into the CH343 port using a win 7 laptop and it won't install a driver. I've held the boot button down until after plugged in but same error. This is a nanoESP32-C6 V1.0 board. What do I need to do to get it to recognize it?

**Comment by iampi31415** [comment]
@mley 

> but I am not sure if the data-pins on these

You most likely need a data cable. I've used the board on MacOS, many cables failed (didn't have a data line), but one worked. In my case both ends are USC-C, but USB-A to USB-C should do as well.
